typedspark/_schema/get_schema_definition.py

```diff
@@ -59,10 +62,21 @@ def _build_schema_definition_string(
             typehint, replace_literals_in=DayTimeIntervalType, replace_literals_by=IntervalType
         )
         if include_documentation:
-            lines += f'    {k}: Annotated[{typehint}, ColumnMeta(comment="")]\n'
+            col_annotated_start = f"    {col_name}: Annotated[{typehint}, "
+            if col_name in schema.__annotations__:
+                if (
+                    hasattr(schema.__annotations__[col_name], "__metadata__")
+                    and schema.__annotations__[col_name].__metadata__ is not None
+                ):
+                    comment == (
+                        col_annotated_start
+                        + f'ColumnMeta(comment="{schema.__annotations__[col_name].__metadata__[0]}")]\n'
+                    )
+                else:
+                    comment = col_annotated_start + 'ColumnMeta(comment="")]\n'
```

Comment:
Why not:
```
comment = schema.__annotations__[col_name].__metadata__[0] 
else:
comment = ""

lines += f"{col_annotated_start}ColumnMeta(comment="{comment}")]\n
```

Reply:
I don't think it is helpful because if `schema.__annotations__[col_name].__metadata__[0]` is None, it will print `'ColumnMeta(comment="None")]'` instead of "", which we don't want. So I still need to have the long if statement

Reply:
```
if (
                    hasattr(schema.__annotations__[col_name], "__metadata__")
                    and schema.__annotations__[col_name].__metadata__ is not None
                ):
comment = schema.__annotations__[col_name].__metadata__[0] 
else:
comment = ""

lines += f"{col_annotated_start}ColumnMeta(comment="{comment}")]\n
```

Just to overcome some code duplication, not a big issues. You could put the big if in a seperate function to make it look better